htmldocx/h2d.py

```diff
@@ -30,6 +32,29 @@
 LIST_INDENT = 0.5
 MAX_INDENT = 5.5 # To stop indents going off the page
 
+def first_child_found_in(parent, tagnames):
+    """
+    Return the first child of parent with tag in *tagnames*, or None if
+    not found.
+    """
+    for tagname in tagnames:
+        child = parent.find(qn(tagname))
+        if child is not None:
+            return child
+    return None
+
+def insert_element_before(parent, elm, successors):
```

Comment:
This is supported by python-docx now, so we don't need to add these helper methods.

Reply:
I removed both of the helper functions.